unit-tests/func/rec-play/test-got-playback-frames.py

```diff
@@ -0,0 +1,245 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#test:device L500*
+#test:device D400*
+
+import pyrealsense2 as rs, os, time, tempfile, sys
+from rspy import devices, log, test
+
+cp = dp = None
+previous_depth_frame_number = -1
+previous_color_frame_number = -1
+got_frames = False
+
+def got_frame():
+    global got_frames
+    got_frames = True
+
+def color_frame_call_back( frame ):
+    global previous_color_frame_number
+    got_frame()
+    test.check_frame_drops( frame, previous_color_frame_number )
+    previous_color_frame_number = frame.get_frame_number()
+
+def depth_frame_call_back( frame ):
+    global previous_depth_frame_number
+    got_frame()
+    test.check_frame_drops( frame, previous_depth_frame_number )
+    previous_depth_frame_number = frame.get_frame_number()
+
+def restart_profiles():
+    """
+    you can't use the same profile twice, but we need the same profile several times. So this function resets the
```

Comment:
Capital.
You can't use the same profile twice? I thought that's exactly what we do in other tests...

Reply:
No, you can't. It raises an error saying the profile is unavailable